### ISSUE TITLE
Stuck into some IMS issues

### ISSUE THREAD

**Issue by modyngs** [herlesupreeth/docker_open5gs#288]
Hi,
Thanks for the project.
I have tried Volte but after a few iteration of calls, the UE wont able to call into another UE!
lets say after about 5 successfull calls, UE wont be able to call the other parties! or just oneway calls ... that needs an Airplain switch and back to cell to be able to continue call to other UEs!

it is happenning alot. I guess there is a state that UE gets out of registration plan while it is connected to the network, but wont able to try call other parties!

Here is the Pcap of what comes in follow:

1- from sam to xiao > declined the call
2- from sam to xiao > answer the call, end on xiao
3-from sam to xiao > answer the call, end on sam
4-from xiao to sam > declined the call 
5-from xiao to sam >  answer the call, end on sam
6-Try to call from xiaomi, but it gets out of the VOLTE sign(maybe kicked out of the network)! while sam is connected and has Volte and network! while xiaomi Volte is gone i can enable internet(4G) sign on it. but not Volte Sign
7- lets turn the Xiaomi on flightmode and then get back it to the network, Now again it has Volte Sign. 
     Lets call from xiao to sam > It is not able to call ... stucks on the calling page on the xiaomi without HD on the called number! but the Volte Sign is still there! ... after few secons (30s) it shows Mobile netowork not available but i still can enable internet (4G sign) and Volte is still there!
     
8- lets call again from xiao to sam > Again the same tthing happened as step 7!
9- lets call from Sam to Xiao > that wont happened eather! stucks on the calling page without HD sign on called number. while after ending the call, UE has Volte sign and internet
10- lets take the Samsung to the airplain and get back to the network. now again it has Volte Sign. lets call from sam to xiao > that call got ended automatically.
11- lets now call from xiao to sam > that wont happed ... stuck on the calling page.
12- lets now take both phones to airplain and give them back to network. first xiao and then sam. 
Now lets call from sam to xiao > declined the call
13- from sam to xiao > answer the call, end on xiao
14- from sam to xiao > answer the call, end on sam
15-from xiao to sam > answer the call, end on sam
16- from xiao to sam >  answer the call, end on xia
17- from xiao to sam >  answer the call ... it called but stuck on answering ... wont transfter sound , ...
18- from xiao to sam >  wont calls , stuck on calling page without HD on called number!
take both phones to airplain and stop the services

Please have a look at this comprehensive test and give me some feedbacks
https://biaupload.com/do.php?filename=org-98006116cadd1.zip (118MB)

**Comment by herlesupreeth** [comment]
Hey, I tried the same at my end using an iPhone and OnePlus phone and didnt have any issue doing 10 calls in a row. May I know which eNB you are using?

**Comment by modyngs** [comment]
@herlesupreeth 
Sure, thats SRS.
UEs are Sam A42 & xiao Mi 11 Lite

**Comment by modyngs** [comment]
@herlesupreeth 
Any idea where can i look for troubleshooting?
Thanks

**Comment by herlesupreeth** [comment]
The call got disconnected in step 6 because the Samsung device sent the BYE not sure why since all the bearers were created correctly. The only other explanation for this behavior could be that radio link is not stable

![image](https://github.com/herlesupreeth/docker_open5gs/assets/4115839/cbb5e5e3-b00c-4a9a-8a0e-f66cdb575639)

And the reason it gets kicked out of the network is because UE is attempting to fallback to CS (which is not there)

![image](https://github.com/herlesupreeth/docker_open5gs/assets/4115839/1b0a1926-0cb5-4610-ab9d-df27712e8c27)

**Comment by modyngs** [comment]
@herlesupreeth 
Thanks for your time
So there might be some wrong stages on the IMS or SIP that disconnected the UE from IMS and now its trying to get into the CS-fallback stages!
Am i Right?
Thanks
Regards

**Comment by herlesupreeth** [comment]
> So there might be some wrong stages on the IMS or SIP that disconnected the UE from IMS and now its trying to get into the CS-fallback stages!
> Am i Right?

No, I dont see any issue from IMS or Core Network. Rather I think the radio link is not stable (SRS)

**Comment by modyngs** [comment]
@herlesupreeth 
any tricks to make it stable?
ideas that causes this issue?

Thanks

**Comment by herlesupreeth** [comment]
I would suggest to do the following:

1. Increase the transmit power of the eNB
2. Try to use a LTE band not used by local operators in your area
3. Run the machine in maximum performance mode so that there no lates/underflows

**Comment by helloTkk** [comment]
> Hey, I tried the same at my end using an iPhone and OnePlus phone and didnt have any issue doing 10 calls in a row. May I know which eNB you are using?

Hi @herlesupreeth, may I know some information (e.g., the exact phone model and android version) about the iPhone and OnePlus phones you use here?
